slides/week1/week1.org

```diff
@@ -130,6 +130,16 @@ Run some demos in your terminal! Ideally you should use each of these commands a
 | =mv= | Move a file |
 | =man= | Access documentation about a command |
 | =grep= | Search for a string |
+| =history= | Shows your command history |
```

Comment:
I think we're going to need to think more about how we do slide layout at some point. I don't really want to bump margins, it helps but it's not a solution.

I split this slide to two for now.